.travis.yml

```diff
@@ -39,8 +39,17 @@ deploy:
     tags: true
     repo: simplabs/ember-simple-auth
 
+after_deploy:
+  - npm i -g now-pipeline
+  - now-pipeline-prune && now-pipeline --alias ember-simple-auth.now.sh
+  - cd server && now-pipeline-prune && now-pipeline --alias ember-simple-auth-server.now.sh
```

Comment:
`after_deploy` should only run after deployments. We can keep an eye on Now and follow up if necessary.